agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -83,7 +83,6 @@
 #include <tlvf/wfa_map/tlvClientCapabilityReport.h>
 #include <tlvf/wfa_map/tlvClientInfo.h>
 #include <tlvf/wfa_map/tlvErrorCode.h>
-#include <tlvf/wfa_map/tlvHigherLayerData.h>
 #include <tlvf/wfa_map/tlvMetricReportingPolicy.h>
```

Comment:
the commit title is still too long (see contribution.md):
you can replace with:
```
agent: move 1905_higher_layer_data handling to son_slave
```

Reply:
The commits message is ok. As long there is no `...` at the end, it is ok.